source/guides/network/supernode-enrollment.rst

```diff
@@ -42,34 +50,36 @@ Symbol Bootstrap is the **easiest way** to create and run |codename| nodes. :doc
 
    .. note:: For test purposes you can use |codename|'s TESTNET using ``-p testnet``.
 
-3. Make sure your node is **sufficiently funded**. Your main account must hold at least **1M** |networkcurrency| :ref:`to be eligible as a supernode <supernode-program>`.
+4. Make sure your node is **sufficiently funded**. Your main account must hold at least **1M** |networkcurrency| :ref:`to be eligible as a supernode <supernode-program>`.
 
-   You can find the node's main account address in the ``target/addresses.yml`` file. 
+   You can find the node's main account address in the ``target/addresses.yml`` file.
 
    On the TESTNET, you can **send tokens** to your node's main account using the `Symbol Faucet <http://faucet.testnet.symboldev.network>`__. If you need more tokens than the faucet can serve, connect to `NEM's Telegram Help Desk <https://t.me/nemhelpdesk>`__ and contact `cryptobeliever <https://t.me/cryptobeliever>`__.
 
-4. **Register the node** by linking its remote, VRF and voting keys:
+5. **Register the node** by linking its remote, VRF and voting keys.
+
+   Without interrupting Symbol Bootstrap, open another terminal and go to the same folder you were before. Then run:
 
    .. code-block:: bash
 
       symbol-bootstrap link --useKnownRestGateways
 
    This sends a transaction linking the supplemental keys to the main account. This transaction pays a small :doc:`fee <../../concepts/fees>` so make sure your main account has **extra funds** beyond the requirements of the Supernode program.
 
-   .. note:: During the test period, heavy testing might artificially increase transaction fees. If ``symbol-bootstrap`` appears to be frozen with the message ``Announcing Simple Transaction hash...`` try increasing the transaction fee with adding a ``--maxFee 10000000`` parameter, for example.
+   .. note:: During the test period, heavy testing might artificially increase transaction fees. If ``symbol-bootstrap`` appears to be frozen with the message ``Announcing Simple Transaction hash...`` try increasing the transaction fee by adding a ``--maxFee 10000000`` parameter, for example (these absolute units corespond to 10 |networkcurrency|).
 
-5. **Enroll in the Supernode program**.
+6. **Enroll in the Supernode program**.
 
    .. code-block:: bash
 
-      symbol-bootstrap enrolRewardProgram --useKnownRestGateways
+      symbol-bootstrap enrollRewardProgram --useKnownRestGateways
 
-   This sends a transaction to the Controller address, which includes the node's public key and its monitoring agent's public URL.
-
-   See the note regarding fees in the previous step.
+   This sends a transaction to the Controller address requesting enrollment (see the note regarding fees in the previous step).
 
    From this point, the :ref:`reward programs controller <reward-programs-controller>` on the network will monitor the node.
```

Comment:
Should we add the following note here as we did for the manual part?

> Once the agent starts running, it will poll the Controller every 2 minutes until the enrollment is completed and new communication certificates are generated. At that point the agent just listens for commands from the Controller.

Reply:
Since this is the "automatic" part of the document I didn't want to give too much details. This is an implementation detail which is taken care of by Bootstrap...